src/node_api.cc

```diff
@@ -706,6 +706,40 @@ bool FindWrapper(v8::Local<v8::Object> obj,
   return true;
 }
 
+static void DeleteEnv(const v8::WeakCallbackInfo<napi_env__>& data) {
+  delete data.GetParameter();
+}
+
+napi_env get_env(v8::Local<v8::Context> context) {
```

Comment:
All other functions in the v8impl namespace uses CamelCase. Might be good to do that here etc.